source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/point/NcaCustomCalculationPoint.java

```diff
@@ -33,8 +36,17 @@ public void setRoadLocalFractionNO2(final Double roadLocalFractionNO2) {
     this.roadLocalFractionNO2 = roadLocalFractionNO2;
   }
 
+  public List<EntityReference> getEntityReferences() {
+    return entityReferences;
+  }
+
+  public void setEntityReferences(final List<EntityReference> entityReferences) {
+    this.entityReferences = entityReferences;
+  }
```

Comment:
Should have mentioned this earlier, but there is a `JsonRoundTripTest` that tests if JSON (de)serializaton works in this module. You could add a new test case for this part (or add it to the existing `NcaCustomCalculationPoint.json` file).